src/plugins/saved_objects_management/public/lib/import_file.ts

```diff
@@ -33,7 +33,10 @@ export async function importFile(
 ) {
   const formData = new FormData();
   formData.append('file', file);
-  const query = createNewCopies ? { createNewCopies } : { overwrite };
+  const query = {
+    createNewCopies,
+    ...(createNewCopies ? { overwrite: false } : { overwrite }),
+  };
```

Comment:
The `createNewCopies` and `overwrite` options cannot be enabled at the same time. Now that `createNewCopies` is enabled by default on the server side, we have to change how these query parameters are constructed.